great_expectations/data_context/templates.py

```diff
@@ -47,18 +47,21 @@
 # used to override and extend Great Expectations.
 plugins_directory: plugins/
 
-expectations_store:
-  # This is where expectations are kept.
-  class_name: ExpectationStore
-  store_backend:
-    class_name: FixedLengthTupleFilesystemStoreBackend
-    base_directory: expectations/
-
+# Stores are configurable places to store things like expectations, evaluation
```

Comment:
Add:
"Three stores are required: expectations, profiling, and evaluation parameters, and must be named here. Additional stores can be configured below and used for data_docs, validation_operators or other purposes"

Reply:
Love it.